github-actions/verify-pr-title/src/index.ts

```diff
@@ -46,7 +46,7 @@ function run(): void {
 				);
 				return;
 			}
-		} else if (baseBranch == "dev" || /^release-\d+\.\d+\.\d+$/.test(baseBranch)) {
+		} else if (baseBranch == "dev" || /^release-\d+\.\d+\.\d+\.*$/.test(baseBranch)) {
```

Comment:
Turns out a lot of the branch names in here were already `dev` and `main`, as opposed to `dev-5`. Which is unfortunate in that it means the enforcement was probably lax in the past, but on the upside, it sure made this pull request cleaner.
This change allows us to properly recognize `release-x.y.z-beta.x` and treat it appropriately.